features/step_definitions/call_volumes_report_steps.rb

```diff
@@ -0,0 +1,44 @@
+Given(/^I am logged in as a Pension Wise data analyst$/) do
+  User.create!(
+    name: 'Data analyst',
+    email: 'analyst@pensionwise.gov.uk',
+    uid: SecureRandom.uuid,
+    permissions: ['signin'],
```

Comment:
Do we want to permit everybody with these permissions or do we need to create a more specific group?